--- a/openid.php
+++ b/openid.php
@@ -30,16 +30,11 @@
 	define( 'OPENID_ENABLE_ADMIN_PANELS', true );
 }
 
-
-$openid_include_path = dirname( __FILE__ ) . '/lib';
-
 // check source of randomness
 if ( ! @is_readable( '/dev/urandom' ) ) {
 	define( 'Auth_OpenID_RAND_SOURCE', null );
 }
 
-set_include_path( $openid_include_path . PATH_SEPARATOR . get_include_path() );
-
 require_once dirname( __FILE__ ) . '/common.php';
 require_once dirname( __FILE__ ) . '/store.php';
 
